Gruntfile.js

```diff
@@ -1,311 +0,0 @@
-    jshint: {
-      files: ["Gruntfile.js"],
-      options: {
-        "curly": true,
-        "eqeqeq": true,
-        "evil": true,
-        "indent": 2,
-        "latedef": true,
-        "globals": {
-          "jQuery": true,
-          "d3": true,
-          "window": true,
-          "console": true,
-          "$": true,
-          "makeRandomData": true,
-          "setTimeout": true,
-          "document": true,
-        },
-        "strict": true,
-        "eqnull": true
-      }
-    },
-    parallelize: {
-      tslint: {
-        all: 4
-      }
-    },
-    clean: {
-      tscommand: ["tscommand*.tmp.txt"]
-    },
-    uglify: {
-      main: {
-        files: {"svgtypewriter.min.js": ["svgtypewriter.js"]}
-      }
-    },
-    "saucelabs-mocha": {
-      all: {
-        options: {
-          urls: ["http://127.0.0.1:9999/test/tests.html"],
-          testname: "SVGTypewriter Sauce Unit Tests",
-          browsers: [{
-            browserName: "firefox",
-            platform: "linux"
-          }, {
-            browserName: "chrome",
-            platform: "linux"
-          }, {
-            browserName: "internet explorer",
-            version: "9",
-            platform: "WIN7"
-          }, {
-            browserName: "safari",
-            platform: "OS X 10.10",
-            version: "8"
-          }],
-          build: process.env.TRAVIS_JOB_ID,
-          "tunnel-identifier": process.env.TRAVIS_JOB_NUMBER
-        }
-      }
-    }
-  };
-
-  var compile_task = [
-    "update_ts_files",
-    "update_test_ts_files",
-    "ts:dev",
-    "tslint",
-    "concat:svgtypewriter",
-    "concat:definitions",
-    "sed:definitions",
-    "sed:private_definitions",
-    "umd",
-    "concat:header",
-    "sed:version_number",
-    "definitions_prod",
-    "test-compile",
-    "clean:tscommand"
-  ];
-
-
-  grunt.initConfig(configJSON);
-  require("load-grunt-tasks")(grunt);
-
-  grunt.registerTask("default", "launch");
-  grunt.registerTask("update_ts_files", updateTsFiles);
-  grunt.registerTask("update_test_ts_files", updateTestTsFiles);
-  grunt.registerTask("definitions_prod", function() {
-    grunt.file.copy("build/svgtypewriter.d.ts", "svgtypewriter.d.ts");
-  });
-
-  grunt.registerTask("dev-compile", compile_task);
-  grunt.registerTask("docs", "typedoc:build");
-  grunt.registerTask("test-sauce", ["connect", "saucelabs-mocha"]);
-  grunt.registerTask("test", [
-    "dev-compile",
-    "blanket_mocha",
-    "parallelize:tslint",
-    "jshint",
-    "ts:verify_d_ts"
-  ]);
-
-  grunt.registerTask("test-compile", [
-    "ts:test",
-    "concat:tests"
-  ]);
-
-  var travisTests = ["test"];
-  if (process.env.SAUCE_USERNAME) {
-    travisTests.push("test-sauce");
-  }
-  grunt.registerTask("test-travis", travisTests);
-
-  grunt.registerTask("dist-compile", [
-    "release-compile",
-    "uglify",
-  ]);
-
-  grunt.registerTask("default", "launch");
-  grunt.registerTask("launch", ["connect", "dev-compile", "watch"]);
```

Comment:
Looks like grunt had a way to spin up a webserver w/ livereload and watch files for changes, but I don't see a way to do this in the new repo?